standalone/default.cfg

```diff
@@ -0,0 +1,393 @@
+
+[DAEMON]
+# set to 1 to run the daemon service within this process;
+# set to 0 if the daemon is run separately (using script joinmarketd.py)
+no_daemon = 1
+
+# Port on which daemon serves; note that communication still
+# occurs over this port even if no_daemon = 1
+daemon_port = 27183
+
+# Currently, running the daemon on a remote host is
+# *NOT* supported, so don't change this variable
+daemon_host = localhost
+
+# by default the client-daemon connection is plaintext, set to 'true' to use TLS;
+# for this, you need to have a valid (self-signed) certificate installed
+use_ssl = false
+
+[BLOCKCHAIN]
+# options: bitcoin-rpc, regtest, bitcoin-rpc-no-history, no-blockchain
+# When using bitcoin-rpc-no-history remember to increase the gap limit to scan for more addresses, try -g 5000
+# Use 'no-blockchain' to run the ob-watcher.py script in scripts/obwatch without current access
+# to Bitcoin Core; note that use of this option for any other purpose is currently unsupported.
+blockchain_source = bitcoin-rpc
+
+# options: signet, testnet, mainnet
+# Note: for regtest, use network = testnet
+network = mainnet
+
+rpc_host = localhost
+# default ports are 8332 for mainnet, 18443 for regtest, 18332 for testnet, 38332 for signet
+rpc_port =
```

Comment:
No, it is empty, because it is like that in the default config as well and intent is to make only minimal changes.
But yes, it can be adapted via environment variables (`jm_rpc_port`).

Reply:
> it can be adapted via environment variables (jm_rpc_port).

It needs to be adapted right? Otherwise it's unset.

Reply:
Yes. Somehow it needs to know how to connect to bitcoin core. I'd refrain from adding a default here. What do you think?